src/main/kotlin/data/persistence/AppPreferences.kt

```diff
@@ -0,0 +1,7 @@
+package data.persistence
+
+// interface to getting user-independent app preferences
+interface AppPreferences {
+    fun getCurrentUserHash() : String // should return an empty string if no current user hash
```

Comment:
I'd recommend just getting/setting the id not the hash

Reply:
fixed